build_tools/docker/dockerfiles/nvidia-bleeding-edge.Dockerfile

```diff
@@ -0,0 +1,67 @@
+# Copyright 2023 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# An image with latest stuff for building IREE using CMake and testing IREE with
+# NVIDIA GPUs.
+
+# To use the host GPUs, `docker run` must be called with the `--gpus all` flag.
+
+# We use .deb files that we host because we have to pin the version and packages
+# routinely dissapear from the Ubuntu apt repositories.
+ARG NVIDIA_GL_DEB="libnvidia-gl-515-server_515.86.01-0ubuntu0.22.04.1_amd64.deb"
+ARG NVIDIA_COMPUTE_DEB="libnvidia-compute-515-server_515.86.01-0ubuntu0.22.04.1_amd64.deb"
+ARG NVIDIA_COMMON_DEB="libnvidia-common-515-server_515.86.01-0ubuntu0.22.04.1_all.deb"
+ARG NVIDIA_EGL_WAYLAND_DEB="libnvidia-egl-wayland1_1.1.9-1.1_amd64.deb"
+
+
+FROM gcr.io/iree-oss/base-bleeding-edge@sha256:31b1896efbb3d1f17aa1a5fe58d9f07ed475c8f099c4b313a590bde9a15097ef AS fetch-nvidia
+ARG NVIDIA_COMMON_DEB
+ARG NVIDIA_GL_DEB
+ARG NVIDIA_COMPUTE_DEB
+ARG NVIDIA_EGL_WAYLAND_DEB
+
+WORKDIR /fetch-nvidia
+RUN wget -q "https://storage.googleapis.com/iree-shared-files/${NVIDIA_COMMON_DEB}"
+RUN wget -q "https://storage.googleapis.com/iree-shared-files/${NVIDIA_GL_DEB}"
+RUN wget -q "https://storage.googleapis.com/iree-shared-files/${NVIDIA_COMPUTE_DEB}"
+RUN wget -q "https://storage.googleapis.com/iree-shared-files/${NVIDIA_EGL_WAYLAND_DEB}"
+
+
+# Set up the image and working directory by inheriting the vulkan CMake
```

Comment:
Huh I think this comment is wrong and wrong in the file this is copied from. This is inheriting from the base image

Reply:
Corrected the comments in both base and bleeding-edge dockerfiles.